README.md

```diff
@@ -87,9 +87,26 @@ In all cases the installation can be validated by running `axiom -v` in the
 terminal:
 
 ```shell
+$ axiom -v
```

Comment:
The description above states `In all cases the installation can be validated by running `axiom -v` in the
 terminal`. I would prefer to not add this line but get rid of the `shell` keyword in the code fence.